### PR TITLE
potion, pokeball, super potion, double colorless, switch reprint system

### PR DESCRIPTION
**Pull request by TheAquaPiplup**
updating potion, pokeball, super potion, double colorless, switch as to make them part of the new reprint system

not done yet

### CODE REVIEW

**Comment by TheAquaPiplup** [comment]
i forgot to put the correct title here. this is legendary collection updates

File: data/src/main/resources/cards/364-phantom_forces.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-schema: E2
+blschema: E2
```
Comment (axpendix) [inline data/src/main/resources/cards/364-phantom_forces.yaml:1]:
```suggestion
schema: E2
```

File: data/src/main/resources/cards/364-phantom_forces.yaml [hunk 2]
```diff
@@ -2698,12 +2698,9 @@ cards:
   name: Double Colorless Energy
   number: '111'
   superType: ENERGY
```
Comment (axpendix) [inline data/src/main/resources/cards/364-phantom_forces.yaml:2700]:
```suggestion
```

**Comment by axpendix** [comment]
Reprint system is getting overhauled soon.

**Pull request closed without merging**